includes/widgets/image-carousel.php

```diff
@@ -92,6 +92,15 @@ public function get_style_depends(): array {
 		return [ 'e-swiper', 'widget-image-carousel' ];
 	}
 
+	public function get_script_depends(): array {
+//		return [ 'import-script-carousel-base' ];
+		return [];
+	}
+
+	public function get_dynamic_imports(): array {
```

Comment:
I am thinking, it might better to use a separate function to list the dynamic imports separate from the script enqueue logic above.